website/static/js/projectorganizer.js

```diff
@@ -417,8 +421,8 @@ function _showProjectDetails(event, item, col) {
                 };
             postAction = $osf.postJSON(url, postData);
             postAction.done(function () {
-                treebeard.updateFolder(null, theParentNode);
-                triggerClickOnItem.call(treebeard, item);    
+                treebeard.updateFolder(null, treebeard.find(1));
```

Comment:
If you are updating the root, which I don't recommend, you don't need to update the parent node.